DisplayOptionsPanel.c

```diff
@@ -20,7 +20,9 @@ in the source distribution for its full text.
 #include "ProvideCurses.h"
 
 
-static const char* const DisplayOptionsFunctions[] = {"      ", "      ", "      ", "      ", "      ", "      ", "      ", "      ", "      ", "Done  ", NULL};
+static const char* const DisplayOptionsFunctions[] = {"Select / Deselect  ","Increment  ", "Decrement  ", "Done  ", NULL};
```

Comment:
Avoid overlong captions
```suggestion
static const char* const DisplayOptionsFunctions[] = {"(De)Select  ","Increment  ", "Decrement  ", "Done  ", NULL};
```

Reply:
Done